src/conf_mode/vpn_openconnect.py

```diff
@@ -132,6 +150,42 @@ def generate(ocserv):
         render(radius_cfg, 'ocserv/radius_conf.tmpl', ocserv["authentication"]["radius"])
         # Render radius servers
         render(radius_servers, 'ocserv/radius_servers.tmpl', ocserv["authentication"]["radius"])
+    elif "local" in ocserv["authentication"]["mode"]:
+        # if mode "OTP", generate OTP users file parameters
+        if "otp" in ocserv["authentication"]["mode"]["local"]:
+            if "local_users" in ocserv["authentication"]:
+                for user in ocserv["authentication"]["local_users"]["username"]:
+                    # OTP token type from CLI parameters:
+                    otp_interval = ocserv["authentication"]["local_users"]["username"][user]["otp"].get("interval", "30")
+                    token_type = ocserv["authentication"]["local_users"]["username"][user]["otp"].get("token-type", "hotp-time")
+                    otp_length = ocserv["authentication"]["local_users"]["username"][user]["otp"].get("otp-length", "6")
```

Comment:
Please merge the defaults using `default` as in this example https://github.com/vyos/vyos-1x/blob/current/src/conf_mode/ssh.py#L52-L53

Reply:
XML "default" doesn't work quite as expected. See comments.

Reply:
Fixed